packages/manager/modules/pci/src/projects/project/private-registry/list/list.html

```diff
@@ -0,0 +1,70 @@
+<div>
+    <oui-page-header
+        data-heading="{{ 'private_registry_title' | translate}}">
+        <!-- guide menu -->
+        <oui-guide-menu data-text="{{ ::'private_registry_guide_main' | translate }}" data-ng-if="false">
+            <oui-guide-menu-item data-text="{{ ::'private_registry_header_guides' | translate }}"
+                                 data-href="{{ ::$ctrl.guideLink }}"
+                                 data-external></oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-page-header>
+
+    <div class="container-fluid px-0 mt-3">
+        <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+        <oui-datagrid class="mt-5" data-rows="$ctrl.registryList.data" data-loading="$ctrl.registryList.loading">
```

Comment:
```suggestion
        <oui-datagrid class="mt-5" data-rows="$ctrl.registryList.data">
```
The loading is now handled with the state changes